app/templates/gulpfile.js

```diff
@@ -294,41 +436,49 @@ gulp.task('optimize', ['inject', 'test'], function() {
   log('Optimizing the js, css, and html');
 
   // Filters are named for the gulp-useref path
-  var cssFilter = $.filter('**/*.css', {restore: true});
-  var jsAppFilter = $.filter('**/' + config.optimized.app, {restore: true});
-  var jslibFilter = $.filter('**/' + config.optimized.lib, {restore: true});
+  var cssFilter = $.filter('**/*.css', {
+    restore: true
+  });
+  var jsAppFilter = $.filter('**/' + config.optimized.app, {
+    restore: true
+  });
+  var jslibFilter = $.filter('**/' + config.optimized.lib, {
+    restore: true
+  });
 
   var templateCache = config.temp + config.templateCache.file;
 
   var combined = gulp
     .src(config.index)
     .pipe($.plumber())
     .pipe(inject(templateCache, 'templates'))
-
```

Comment:
Hmm my beautifier doesn't really like it when I try and doesn't align all the methods properly